### PR TITLE
Add Appraisal

### PR DESCRIPTION
**Pull request by openbl**
Tests pass with the following Rails dependencies:
- [x] ~~4.0~~
- [x] 4.1
- [x] 4.2

### CODE REVIEW

**Comment by openbl** [comment]
I don't think Rails 4.0 needs supporting since that version is itself no longer supported.

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /.bundle/
 /.yardoc
-/Gemfile.lock
+*emfile.lock
```
Comment (jcoleman) [inline .gitignore]: I assume appraisal generates something like `/rails-4.1.gemfile.lock`?

Could we tighten the matching up here?

Comment (openbl) [inline .gitignore]: It generates `./gemfiles/rails_4_1.gemfile.lock`. I don't feel strongly either way. I could change it to be the top-level `./Gemfile.lock` and `./gemfiles/*gemfile.lock`

Comment (jcoleman) [inline .gitignore]: That would be great!

**Comment by jcoleman** [comment]
After the one change comment happens, I'd like to see this as two commits: one to add appraisal and one to remove Rails 4.0 support. Otherwise looks good.

**Pull request merged**